build.gradle.kts

```diff
@@ -45,6 +45,22 @@ allprojects {
         create<BasicAuthentication>("basic")
       }
     }
+    // Allow Mapbox snapshot maven if a release is **not** being built
+    val releaseTagRegex = "^v[0-9]+\\.[0-9]+\\.[0-9]+.*\$".toRegex()
+    val circleTag = System.getenv("CIRCLE_TAG") ?: ""
+    if (!releaseTagRegex.matches(circleTag)) {
```

Comment:
Maybe rewrite it a little bit : 
```
    val releaseIsBuilt = System.getenv("CIRCLE_TAG").matches(circleTag) ?: false
    if (releaseIsBuilt) {
```
Then comment will be unneeded

Reply:
Thanks! see bf3d2acf731502c749d7647f0cd4e341a8e93dff